tools/send-to-caas/send-utils.js

```diff
@@ -143,9 +143,16 @@ const prefixHttps = (url) => {
   return url;
 };
 
+const flattenLink = (link) => {
+  const htmlElement = document.createElement('div');
+  htmlElement.innerHTML = link;
+  return htmlElement.querySelector('a').getAttribute('href');
+};
+
 const checkUrl = (url, errorMsg) => {
   if (url === undefined) return url;
-  return isValidUrl(url) ? prefixHttps(url) : { error: errorMsg };
+  const flatUrl = (url.indexOf('href=')) ? flattenLink(url) : url;
```

Comment:
`-1` would eval to true

```suggestion
  const flatUrl = url.includes('href=') ? flattenLink(url) : url;
```